scripts/mk-docs.sh

```diff
@@ -25,7 +25,7 @@ pushd "${PROJECT_ROOT}" > /dev/null 2>&1
       --crate-version ${DOC_VERSION} \
       --enable-index-page \
       --generate-link-to-definition" \
-      cargo doc -Zrustdoc-map --no-deps --all-features \
+      cargo +nightly doc -Zrustdoc-map --no-deps --all-features \
```

Comment:
Is this actually necessary?

Reply:
The `-Z` flags are exclusive to Rust nightly. You might have set up a directory override on your machine (and iirc, the workflow also changes the default for the directory).

I think it's marginally more convenient for running it locally.